docs/administrators_guide.md

```diff
@@ -0,0 +1,17 @@
+# Administrators Guide
+
+This document describes useful information when administering a bridge. 
+
+## Scaling
+
+TODO: This
+
+## Hot Reloading
+
+The bridge supports hot-reloading of the configuration file by sending a `SIGHUP` signal. Some configuration keys will
```

Comment:
It would be cool to have a code sample on how to send a SIGHUP signal on Linux/OSX.

Reply:
Done.